cdc/sink/producer/kafka/kafka.go

```diff
@@ -354,17 +354,18 @@ func validateMaxMessageBytesAndCreateTopic(admin kafka.ClusterAdminClient, topic
 	info, exists := topics[topic]
 	// once we have found the topic, no matter `auto-create-topic`, make sure user input parameters are valid.
 	if exists {
-		// make sure that topic's `max.message.bytes` is not less than given `max-message-bytes`
-		// else the producer will send message that too large to make topic reject, then changefeed would error.
+		// make sure that producer's `MaxMessageBytes` smaller than topic's `max.message.bytes`
 		topicMaxMessageBytes, err := getTopicMaxMessageBytes(admin, info)
 		if err != nil {
 			return cerror.WrapError(cerror.ErrKafkaNewSaramaProducer, err)
 		}
+
 		if topicMaxMessageBytes < config.MaxMessageBytes {
-			return cerror.ErrKafkaInvalidConfig.GenWithStack(
-				"topic already exist, and topic's max.message.bytes(%d) less than max-message-bytes(%d). "+
-					"Please make sure `max-message-bytes` not greater than topic's `max.message.bytes`",
-				topicMaxMessageBytes, config.MaxMessageBytes)
+			log.Warn("topic's `max.message.bytes` less than user set `max-message-bytes`,"+
+				"use topic's `max.message.bytes` to initialize the Kafka producer",
+				zap.Int("max.message.bytes", topicMaxMessageBytes),
```

Comment:
```suggestion
				zap.Int("message.max.bytes", topicMaxMessageBytes),
```

Reply:
for the topic, it's `max.message.bytes`, for the broker, it's `message.max.bytes`

Reply:
Got it, I found it in Kafka document, it's strange.